Sources/ViewInspector/Inspector.swift

```diff
@@ -110,10 +110,15 @@ internal extension Inspector {
 private extension String {
     func sanitizingNamespace() -> String {
         var str = self
-        while let range = str.range(of: ".(unknown context at ") {
-            let end = str.index(range.upperBound, offsetBy: .init(11))
-            str.replaceSubrange(range.lowerBound..<end, with: "")
-        }
+
+        let pattern = "\\.\\(unknown context at ..........\\)"
+        let regex = try! NSRegularExpression(pattern: pattern)
+        let range = NSRange(location: 0, length: str.utf16.count)
```

Comment:
@rafael-assis can you help me understand why we chose `str.utf16.count` vs. `str.count` or `str.utf8.count`? In my experience UTF16 encoding is less common than UTF8. Additionally, in examples I've found online I'm seeing the equivalent of `str.count` ([1](https://stackoverflow.com/a/46899134), [2](https://medium.com/@dkw5877/regular-expressions-in-swift-928561ad55c8)).

Reply:
Thanks for the detailed response. `str.utf16.count` seems to be correct. Wow, TIL!

<img width="1512" alt="Screenshot 2023-10-11 at 1 14 25 PM" src="https://github.com/nalexn/ViewInspector/assets/1791049/6643545c-94fb-4df8-9f98-31c0aae9398d">